Topics/Software_Tools/fish.md

```diff
@@ -0,0 +1,80 @@
+# A fin-troduction to `fish`
+
+## Introduction
+
+### enter `fish`!
+
+...the **f**riendly **i**nteractive **sh**ell.
+
+fish offers a command-line interface that is focused on usability and interactive use[^1].
+
+This introduction will be less about the syntax differences between the fish language
+and bash or zsh and more about the out-of-the-box features that are offered from this shell.
+
```

Comment:
You should briefly define what a shell is.

Reply:
Thanks for the suggestion. I'll add a brief definition.